### PR TITLE
Merge 18.7.0.1 (beta 2) into `develop`

### PR DESCRIPTION
**Pull request by mokagio**
I made this to ship a build with the most up-to-date translations we have to help smoke testing them in the app.

Includes:

- A CoreData fix https://github.com/wordpress-mobile/WordPress-iOS/pull/17499 by @twstokes 
- A fix for the vertical alignment of the Reply view https://github.com/wordpress-mobile/WordPress-iOS/pull/17491 by @ScoutHarris 
- A fix for the vertical alignment of the Empty view in the Comments screen https://github.com/wordpress-mobile/WordPress-iOS/pull/17502 by @ScoutHarris 

Also notice the new subtitles for Jetpack, and that quite a few of them go over the 30 characters limit 😞 

The request for the translator is visible in GlotPress, albeit not "in your face". It would be nice if we could have some kind of annotation in the `.po` file that GlotPress can read and use to enforce a characters threshold.

Screen Shot 2021-11-25 at 2 47 53 pm

### CODE REVIEW

File: fastlane/jetpack_metadata/ar-SA/subtitle.txt [hunk 1]
```diff
@@ -1 +1 @@
-اجعل موقعك على ووردبريس أسرع وأكثر أمانًا.
\ No newline at end of file
+ووردبريس أسرع وأكثر أمانًا
```
Comment (mokagio) [inline fastlane/jetpack_metadata/ar-SA/subtitle.txt:1]: 26 characters ✅

File: fastlane/jetpack_metadata/de-DE/subtitle.txt [hunk 2]
```diff
@@ -1 +1 @@
-Mache deine WordPress-Website schneller und sicherer.
\ No newline at end of file
+WordPress – schneller und sicherer
```
Comment (mokagio) [inline fastlane/jetpack_metadata/de-DE/subtitle.txt:1]: 34 characters ❌

File: fastlane/jetpack_metadata/es-ES/subtitle.txt [hunk 3]
```diff
@@ -1 +1 @@
-Mejora la velocidad y la seguridad de tu sitio de WordPress.
\ No newline at end of file
+WordPress más rápido y seguro
```
Comment (mokagio) [inline fastlane/jetpack_metadata/es-ES/subtitle.txt:1]: 29 characters ✅

File: fastlane/jetpack_metadata/fr-FR/subtitle.txt [hunk 4]
```diff
@@ -1 +1 @@
-Site WP plus sûr, plus rapide.
\ No newline at end of file
+WordPress plus sûr, plus rapide
```
Comment (mokagio) [inline fastlane/jetpack_metadata/fr-FR/subtitle.txt:1]: 31 characters ❌

File: fastlane/jetpack_metadata/he/subtitle.txt [hunk 5]
```diff
@@ -1 +1 @@
-כדאי לוודא שהאתר שלך ב-WordPress עובד בצורה במהירה והבטוחה ביותר.
\ No newline at end of file
+Faster, safer WordPress
```
Comment (mokagio) [inline fastlane/jetpack_metadata/he/subtitle.txt:1]: This obviously hasn't been translated yet

File: fastlane/jetpack_metadata/id/subtitle.txt [hunk 6]
```diff
@@ -1 +1 @@
-Buat situs WordPress Anda lebih cepat dan lebih aman.
\ No newline at end of file
+WordPress lebih cepat dan aman
```
Comment (mokagio) [inline fastlane/jetpack_metadata/id/subtitle.txt:1]: 30 characters ✅, just 😅

File: fastlane/jetpack_metadata/it/subtitle.txt [hunk 7]
```diff
@@ -1 +1 @@
-Rendi più sicuro e più rapido il tuo sito WordPress.
\ No newline at end of file
+WordPress più veloce e più sicuro
```
Comment (mokagio) [inline fastlane/jetpack_metadata/it/subtitle.txt:1]: 33 characters ❌

I'll update in GlotPress to "WordPress più veloce e sicuro" which means the same in Italian and is 29 characters.

File: fastlane/jetpack_metadata/ja/subtitle.txt [hunk 8]
```diff
@@ -1 +1 @@
-WordPress サイトをもっと速く、もっと安全に。
\ No newline at end of file
+より速く安全な WordPress
```
Comment (mokagio) [inline fastlane/jetpack_metadata/ja/subtitle.txt:1]: 17 characters ✅ 

That is if App Store Connect does something like `"...".length`, which would make sense to me given that the 30 characters limitation is not a matter of sentence complexity but of screen real estate. I'm not actually sure if Japanese strings kanji are counted in the same way as we do with the letters in a Roman language.

File: fastlane/jetpack_metadata/ko/subtitle.txt [hunk 9]
```diff
@@ -1 +1 @@
-WordPress 사이트를 더 빠르고 더 안전하게 만드세요.
\ No newline at end of file
+더 빠르고 더 안전한 워드프레스
```
Comment (mokagio) [inline fastlane/jetpack_metadata/ko/subtitle.txt:1]: 17 characters ✅

That is if App Store Connect does something like "...".length, which would make sense to me given that the 30 characters limitation is not a matter of sentence complexity but of screen real estate. I'm not actually sure if Korean characters are counted in the same way as we do with the letters in a Roman language.

File: fastlane/jetpack_metadata/nl-NL/subtitle.txt [hunk 10]
```diff
@@ -1 +1 @@
-Maak je WordPress-site sneller en veiliger.
\ No newline at end of file
+Sneller, veiliger WordPress
```
Comment (mokagio) [inline fastlane/jetpack_metadata/nl-NL/subtitle.txt:1]: 27 characters ✅

File: fastlane/jetpack_metadata/pt-BR/subtitle.txt [hunk 11]
```diff
@@ -1 +1 @@
-Deixe seu site do WordPress mais rápido e seguro.
\ No newline at end of file
+O WordPress mais ágil e seguro
```
Comment (mokagio) [inline fastlane/jetpack_metadata/pt-BR/subtitle.txt:1]: 30 characters ✅

File: fastlane/jetpack_metadata/ru/subtitle.txt [hunk 12]
```diff
@@ -1 +1 @@
-Сделайте свой сайт WordPress быстрее и безопаснее.
\ No newline at end of file
+Более быстрый и безопасный WordPress
```
Comment (mokagio) [inline fastlane/jetpack_metadata/ru/subtitle.txt:1]: 36 characters ❌

File: fastlane/jetpack_metadata/sv/subtitle.txt [hunk 13]
```diff
@@ -1 +1 @@
-Gör din WordPress-webbplats snabbare och säkrare.
\ No newline at end of file
+Snabbare, säkrare WordPress
```
Comment (mokagio) [inline fastlane/jetpack_metadata/sv/subtitle.txt:1]: 27 characters ✅

File: fastlane/jetpack_metadata/tr/subtitle.txt [hunk 14]
```diff
@@ -1 +1 @@
-WordPress sitenizi daha hızlı ve daha güvenli hale getirin.
\ No newline at end of file
+Daha hızlı, daha güvenli bir WordPress
```
Comment (mokagio) [inline fastlane/jetpack_metadata/tr/subtitle.txt:1]: 38 characters ❌

File: fastlane/jetpack_metadata/zh-Hans/subtitle.txt [hunk 15]
```diff
@@ -1 +1 @@
-提高您 WordPress 站点的速度和安全性。
\ No newline at end of file
+更快速、更安全的 WordPress
```
Comment (mokagio) [inline fastlane/jetpack_metadata/zh-Hans/subtitle.txt:1]: 18 characters ✅

That is if App Store Connect does something like "...".length, which would make sense to me given that the 30 characters limitation is not a matter of sentence complexity but of screen real estate. I'm not actually sure if Chinese characters are counted in the same way as we do with the letters in a Roman language.

File: fastlane/jetpack_metadata/zh-Hant/subtitle.txt [hunk 16]
```diff
@@ -1 +1 @@
-讓你的 WordPress 網站更快更安全。
\ No newline at end of file
+更快速安全的 WordPress 體驗
```
Comment (mokagio) [inline fastlane/jetpack_metadata/zh-Hant/subtitle.txt:1]: 19 characters ✅

That is if App Store Connect does something like `"...".length`, which would make sense to me given that the 30 characters limitation is not a matter of sentence complexity but of screen real estate. I'm not actually sure if Chinese characters are counted in the same way as we do with the letters in a Roman language.

**Review by mokagio — COMMENTED** [review]
- [x] Updates to the localization files (`.strings` and Fastlane metadata) automatically pulled for the new translations that have already been approved in GlotPress
- [x] Version update in `.xcconfig`
- [x] Diffs from the PRs that made it into this beta

File: fastlane/jetpack_metadata/zh-Hant/subtitle.txt (hunk 16 above)
Comment (AliSoftware) [reply fastlane/jetpack_metadata/zh-Hant/subtitle.txt:1]: Can confirm that's how they're counted 🙂 

image

**Review by AliSoftware — APPROVED** [review]
Looking good.

Approving despite some subtitles being >30 because you already made a request in +i18n to ask them to be fixed by translators (ref: pxLjZ-6GO-p2) so best scenario they're gonna be fixed by the time you submit on Friday, and worst case you'll just skip updating them this time around and hope they get fixed for the next.

PS: Edited the PR's title which was mentioning `18.7.0.2` while this is `18.7.0.1` (beta 2, but zero-based versioning, yadda yadda)

File: fastlane/jetpack_metadata/zh-Hant/subtitle.txt (hunk 16 above)
Comment (mokagio) [reply fastlane/jetpack_metadata/zh-Hant/subtitle.txt:1]: Oh, thank you for checking ❤️ `TIL`

**Comment by mokagio** [comment]
> PS: Edited the PR's title which was mentioning 18.7.0.2 while this is 18.7.0.1 (beta 2, but zero-based versioning, yadda yadda)

Lol. Yeah I should have actually called this beta 1 (with code freeze being beta 0). I think in my mind doing the first beta on the Thursday before finalization was just too weird and my subconscious just assumed this was another one 🤣 

Thanks for fixing it.

**Pull request merged**